### DISCUSSION TITLE
Reward Distribution January 2021 and The Closing of Developer Program Rewards

### DISCUSSION THREAD

**Discussion by ngkong** [Tadpole-finance/dev-rewards-program#8] [General]
Hi fellow developers,

This week we distribute the Developer Rewards for the period of January 2021. Based on the calculated using the latest rules, no one but @rajacebong are eligible for this month's rewards.

**The Closing of Developer Program Rewards**

We created the Developer Rewards Program to motivate the open-source community to participate in building DeFi protocols. This program's main objective is to move forward driven by the open-source community where the Tadpole Finance core team acts as a guide and moderator. In this way, our intention is for this project to be sustainable and purely driven by the community. 

Currently, the Developer Rewards Program has been running for two months. To date, there are 15 merged PRs at Tadpole-finance/tadpole-finance.github.io; 10 merged PR at Tadpole-finance/docusaurus; and 0 merged PR in Tadpole-finance/Tadpole-finance. In Tadpole-finance/Tadpole-finance there are only 2 PR requests, which both of them are rejected. When observed from the layers, almost all developments are done at the front-end layer. There is no progress whatsoever on the smart contract side of the Tadpole Protocol. 

Based on the above analysis, we conclude that this program is not effective enough in motivating developers to develop the DeFi smart contract protocol. Especially when compared to the rewards distributed, which is 8333 TAD per month or more than 100,000 USD when converted using the latest TAD prices. Tadpole Finance can actually spend the same amount of cost on hiring 50-100 programmers full-time. Meanwhile, we only get 25 approved PRs, and none of them are at the smart contract level.

Based on these reasons, we decided to end the Developer Rewards program and replace it with a selected developer team, which we will announce in the future. This program ends at the beginning of February 2021. The last distribution of rewards is for the period January 2021. All remaining TAD set aside for the Developer Rewards Program will be kept and distributed regularly to selected developers for further project development. 

To all the developers who already participate in the Developer Rewards Program, we thank you for your contributions. If you are interested in joining as a Tadpole Finance full-time programmer, please contact @ngkong or let's meet at the telegram group https://t.me/TadpoleFinanceDev

**Comment by edosetiawan-xyz** [comment]
Saya mau tanya, apakah dg suplai tad yg ada 1.jt bisa berpotensi harga seperti bitcoin.
